### PR TITLE
observerward: add observer_ward alias

### PR DESCRIPTION
**Pull request by cn-kali-team**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/observer_ward.rb [hunk 1]
```diff
@@ -15,8 +15,13 @@ class Observerward < Formula
   end
 
   depends_on "rust" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl@1.1"
 
   def install
+    # Ensure that the `openssl` crate picks up the intended library.
+    # https://crates.io/crates/openssl#manual-configuration
+    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix
```
Comment (SMillerDev) [inline Formula/observer_ward.rb:24]: This is only needed on Linux since macOS LibreSSL is also supported

**Review by SMillerDev — COMMENTED** [review]
It seems the rename might just as well be an alias since it doesn't actually change the name significantly.

**Review by carlocab — COMMENTED** [review]
The project is called `ObserverWard` so `observerward` is fine as the name. You can add an alias for `observer_ward` if you prefer.

**Comment by cn-kali-team** [comment]
- Can I write an automatic update action like this?
```bash
brew install observer_ward
```
https://github.com/0x727/ObserverWard/blob/8fca9b475e7b1de50968322e8d8082e7e4d07858/.github/workflows/post-release.yml#L158

**Comment by cn-kali-team** [comment]
Sorry, my account was wrongly marked before. It returned to normal yesterday. Can I merge now?

**Review by Bo98 — APPROVED** [review]

**Pull request merged**